src/adaptors/http2/http2_adaptor.c

```diff
@@ -2426,6 +2436,8 @@ static void handle_connection_event(pn_event_t *e, qd_server_t *qd_server, void
             send_settings_frame(conn);
             qd_log(log, QD_LOG_INFO, "[C%"PRIu64"] Accepted Ingress ((PN_RAW_CONNECTION_CONNECTED)) from %s", conn->conn_id, conn->remote_address);
         } else {
+            sys_atomic_init(&conn->raw_closed_read, 0);
```

Comment:
Why re-init?  Won't this destroy the existing state?

Reply:
Re-init was wrong. Now uses CLEAR_ATOMIC_FLAG